pkgs/desktops/gnome-3/core/tracker-miners/default.nix

```diff
@@ -51,6 +84,18 @@ in stdenv.mkDerivation rec {
     };
   };
 
+  postPatch = ''
+    substituteInPlace src/tracker-extract/10-flac.rule --replace audio/x-flac audio/flac
+    substituteInPlace src/tracker-extract/meson.build \
+      --replace tracker_common_dep tracker_miners_common_dep
+  '';
+
+  postFixup = ''
+    ${glib.dev}/bin/glib-compile-schemas $out/share/gsettings-schemas/${name}/glib-2.0/schemas
```

Comment:
Schema compilation traditionally belongs to `postInstall`.

Reply:
Moving to `postInstall` causes this error:

```
Error opening directory “/nix/store/xck0468mbapjd98bg9dvavwx5gfyw80c-tracker-miners-2.0.4/share/gsettings-schemas/tracker-miners-2.0.4/glib-2.0/schemas”: No such file or directory
```

So, this path doesn't exist until `fixupPhase` :-(

Reply:
Yes, it is moved in `preFixup` by glib setup hook. You would use `glib-compile-schemas $out/share/glib-2.0/schemas`.